models/spleen_ct_segmentation/configs/metadata.json

```diff
@@ -1,7 +1,8 @@
 {
     "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20220324.json",
-    "version": "0.5.0",
+    "version": "0.5.1",
     "changelog": {
+        "0.5.1": "modify to show test process",
```

Comment:
I don't see you changed anything in this bundle?

Thanks.

Reply:
it's used to trigger the test to show the running result. Will revert the changes of this bundle after the PR is ready to merge.